server_installation/topics/operator/external-database.adoc

```diff
@@ -36,6 +37,21 @@ The other properties work in the same way for a hosted or external database. Set
 * `POSTGRES_USERNAME` - Database username
 * `POSTGRES_PASSWORD` - Database password
 * `POSTGRES_SUPERUSER` - Indicates, whether backups should run as super user. Typically `true`.
+* `SSL_MODE` - Indicates, whether to use TLS on the connection to the external PostgreSQL database, check the possible https://www.postgresql.org/docs/current/libpq-ssl.html[values]
```

Comment:
@andymunro you suggested to remove the comma after Indicates .... should I remove it too in the previous line ( it was there before and copied the style from it ) ?